src/js/modules/infragistics.ui.editors.js

```diff
@@ -4055,7 +4055,7 @@
 				```
 				*/
 			maxDecimals: null,
```

Comment:
@Lipata Actually I see a set in the snippet - so this change should probably also address the behavior when setting this options with an existing value.